symfony/src/Entity/DeceasedReport.php

```diff
@@ -0,0 +1,307 @@
+<?php
+
+namespace App\Entity;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class DeceasedReport
```

Comment:
Even though this is an API resource, it made sense to me to encapsulate it in a class for easier portability (and less code repetition).